.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,49 @@
+---
+name  : Bug report
+about : Report a bug in Shiny.
+---
+
+<!--
+
+Before you file an issue, we would appreciate it if you would upgrade to the latest version of Shiny from CRAN and confirm that the problem persists.
+
+# To install latest shiny from CRAN:
+install.packages("shiny")
+# To update all your R packages:
+update.packages(ask = FALSE, checkBuilt = TRUE)
+# Please remember to restart R after updating packages.
+
+This issue tracker is for bugs and feature requests in the Shiny package. If you're having trouble with Shiny Server or a related package, please file an issue in the appropriate repository. 
+
+If you're having trouble with shinyapps.io, and you have a paid account (Starter, Basic, Standard, or Pro), please file a support ticket via https://support.rstudio.com. If you have a Free account, please post to the RStudio Community with the shinyappsio tag: https://community.rstudio.com/tags/shinyappsio.
+
+Finally, if you are an RStudio customer and are having trouble with one of our Pro products, get in touch with our support team at support@rstudio.com.
+
+See our guide to writing good bug reports for further guidance: https://github.com/rstudio/shiny/wiki/Writing-Good-Bug-Reports. The better your report is, the likelier we are to be able to reproduce and ultimately solve it.
+-->
+
+### System details
+
+    Shiny Version           :
+    OS Version              : 
+    R Version               : 
+    Browser Version         :
+    RStudio Edition/Version : <!-- If applicable -->
+
+### Example application *or* steps to reproduce the problem
+
+<!-- A minimal, self-contained example app demonstrating the problem is extremely helpful to us. Thank you in advance for taking the time to create one. For guidance on how to do so, please see https://github.com/rstudio/shiny/wiki/Writing-Good-Bug-Reports -->
```

Comment:
I think we should copy the content from the "Creating Shiny reprexes" from https://community.rstudio.com/t/shiny-debugging-and-reprex-guide/10001 to the wiki page. I don't think that people are going to follow two links to get there.

Also, I think that it would be good if the wiki page itself were more concise because I don't think many people will want to take the time to read all of it.